ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -995,6 +1001,41 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
     self.hairline.alpha = alpha;
 }
 
+- (NSArray *)stepsForReviewStep:(ORKReviewStep *)reviewStep {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    if (reviewStep.isStandalone) {
+        for (ORKStep *step in reviewStep.steps) {
+            [steps addObject:step];
+        }
+    } else {
+        ORKStep *nextStep = reviewStep;
```

Comment:
2 main reasons: (1) using `_managedStepIdentifiers` would result in a subsequent call to `stepWithIdentifier:`, which would limit the functionality to tasks that implement this method
(2) Imagine a task with 3 steps, first one is an embedded review step with `reverseListing`, second and third one are question steps. Since `_managedStepIdentifiers` only contains steps that have been visited/completed, the review step wouldn't discover steps 2 and 3.